android/src/main/java/com/oney/WebRTCModule/DailyAudioManager.java

```diff
@@ -209,77 +204,37 @@ private void abandonAudioFocus() {
 
     private void configureDevicesForCurrentMode() {
         Log.d(TAG, "configureDevicesForCurrentMode => " + mode);
+        if(mode == Mode.USER_DEFINED){
+            return; // we don't influentiate if the user is commanding the device selection
```

Comment:
Nit: comment is unclear - maybe "If the user is manually controlling device selection, don't do anything"

Reply:
Fixed